install.sh

```diff
@@ -714,6 +715,10 @@ systemd_enable() {
 systemd_start() {
   info "systemd: Starting node_exporter"
   $SUDO systemctl restart node_exporter
+  # Wait an arbitrary amount of time for service to load (systemctl status would report successful start while starting)
+  sleep 1
+  $SUDO systemctl status node_exporter > /dev/null 2>&1
```

Comment:
I do think fatal is the way to go, since, node_exporter is installed of course, but isn't installed properly and won't work anytime soon. So the whole point of the script (installing and configuring node_exporter) fails, which IMHO is a fatal level.